app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
 
+    people = {}
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
+
+
+def create_person_list(people: list) -> list[Person]:
+    new_people_list = []
+
+    for human in people:
+        new_people_list.append(Person(human["name"], human["age"]))
+
+    for human in people:
+        if "wife" in human and human["wife"]:
```

Comment:
Use the `dict.get()` method instead of explicitly checking for the presence of a key. This approach simplifies the code and avoids potential `KeyError` exceptions, making it cleaner and more concise. For example: `if human.get('wife'):`